Artsiom_Dziavitski/12.Kubernetes.First_deployment/manifests/self-hosted-runner/01-cert-manager.yaml

```diff
@@ -0,0 +1,5388 @@
+# Copyright 2021 The cert-manager Authors.
```

Comment:
looks like to many for one runner, it is very good you find this solution, but to many resources, manifests for one runner

Reply:
please do not deploy on our servers, too heavy for them